binary-search/brute-force-solution.js

```diff
@@ -0,0 +1,30 @@
+const testCases = [
+  {
+    arrayNumbers: [2, 4, 7, 7, 9],
+    queries: [7, 10, 4, 2],
+  },
+  {
+    arrayNumbers: [1, 3, 5, 7, 9, 11],
+    queries: [3, 6, 8, 11],
+  },
+];
+
+function checkNumberIndex(listOfNumbers, listOfQueries) {
+  let resultsQueries = []
+  for(let j = 0; j < listOfQueries.length; j++){
+    let elementIndex = -1;
+    resultsQueries.push(elementIndex)
```

Comment:
ser vc mudar a inserção da resposta pra depois do for, vc não vai precisar modificar a lista de respostas caso encontre o valor

Reply:
Boa mano. Valeu fica mais enxuto!